tests/integration/suite/test_pod_security_policies.py

```diff
@@ -43,6 +43,25 @@ def create_pspt(client):
     return pspt
 
 
+def set_psp_enabled(k8s_dynamic_client, value):
```

Comment:
Any test that uses this function should also have a fixture that sets the local cluster back to what it was before the tests started

Reply:
Refer to the fixture `restore_rancher_version` as a good example